freeform_json_to_yaml.py

```diff
@@ -0,0 +1,440 @@
+import yaml, argparse, json
+from random import randint
+from logger import LogLevel, Logger
+
+LOGGER = Logger('JsonConverter')
+
+class Indenter(yaml.Dumper):
+    ### Source: https://stackoverflow.com/questions/25108581/python-yaml-dump-bad-indentation#:~:text=class%20MyDumper(yaml.Dumper)%3A%0A%0A%20%20%20%20def%20increase_indent(self%2C%20flow%3DFalse%2C%20indentless%3DFalse)%3A%0A%20%20%20%20%20%20%20%20return%20super(MyDumper%2C%20self).increase_indent(flow%2C%20False)
+    def increase_indent(self, flow=False, indentless=False):
+        return super(Indenter, self).increase_indent(flow, False)
+
+SUPPLY_MAP = {
+    "hemostaticGauzeCount": "Hemostatic gauze",
+    "tourniquetCount": "Tourniquet",
+    "pressureBandageCount": "Pressure bandage",
+    "decompressionNeedleCount": "Decompression Needle",
+    "nasopharyngealAirwayCount": "Nasopharyngeal airway",
+    "pulseOximeterAvailable": "Pulse Oximeter",
+    "painMedsCount": "Pain Medications",
+    "splintCount": "Splint",
+    "ivBloodCount": "Blood",
+    "ivSalineCount": "IV Bag",
+    "burnDressingCount": "Burn Dressing",
+    "epiPenCount": "Epi Pen",
+    "chestSealCount": "Vented Chest Seal",
+    "blanketCount": "Blanket",
+    "lollipopCount": "Fentanyl Lollipop"
+}
+
+MENTAL_STATUS_MAP = {
+    "agony": "AGONY",
+    "calm": "CALM",
+    "unresponsive": "UNRESPONSIVE",
+    "dead": "UNRESPONSIVE",
+    "upset": "UPSET"
+}
+
+BREATHING_MAP = {
+    "normal": "NORMAL",
+    "fast": "FAST",
+    "restricted": "RESTRICTED",
+    "none": "NONE",
+    "collapsedLeft": "RESTRICTED",
+    "collapsedRight": "RESTRICTED",
+    "collapsedBoth": "RESTRICTED"
+}
+
+PULSE_MAP = {
+    "none": "NONE",
+    "faint": "FAINT",
+    "normal": "NORMAL",
+    "fast": "FAST"
+}
+
+INJ_LOC_MAP = {
+    "R Forearm": "right forearm",
+    "L Forearm": "left forearm",
+    "R Calf": "right calf",
+    "L Calf": "left calf",
+    "R Thigh": "right thigh",
+    "L Thigh": "left thigh",
+    "R Stomach": "right stomach",
+    "L Stomach": "left stomach",
+    "R Bicep": "right bicep",
+    "L Bicep": "left bicep",
+    "R Shoulder": "right shoulder",
+    "L Shoulder": "left shoulder",
+    "R Side": "right side",
+    "L Side": "left side",
+    "R Chest": "right chest",
+    "C Chest": "center chest",
+    "L Chest": "left chest",
+    "R Wrist": "right wrist",
+    "L Wrist": "left wrist",
+    "Face": "left face",
+    "L Neck": "left neck",
+    "R Neck": "right neck",
+    "R Leg": "right leg",
+    "L Leg": "left leg",
+    "R Palm": "right hand",
+    "L Palm": "left hand",
+    "R Upper Leg": "right thigh",
+    "L Upper Leg": "left thigh",
+    "R Shin": "right calf",
+    "L Shin": "left calf",
+    "L Lower Leg": "left calf",
+    "R Lower Leg": "right calf"
```

Comment:
That's a good point. I don't know how that's covered in the yaml. I was pulling from the api to get this list since that's what we're trying to match. @nextcen-dgemoets how are full/l/r body burns created using the api? Should I list every possible burn?

Reply:
Right, the API has no notion of "full body" or "entire left/right side".   I guess we just map it ourselves and create multiple burn injuries?

Reply:
I think I've covered this now